src/yunohost/settings.py

```diff
@@ -39,6 +39,7 @@
     ("security.password.admin.strength", {"type": "int", "default": 1}),
     ("security.password.user.strength", {"type": "int", "default": 1}),
     ("service.ssh.allow_deprecated_dsa_hostkey", {"type": "bool", "default": False}),
+    ("security.ciphers.compatibility", {"type": "string", "default": "intermediate"}),
```

Comment:
I think we should be using a choice here ? e.g. `"choices": ["intermediate", "modern"]` (c.f. "example.enum" a few line up)

Reply:
Definitely !